packages/react/src/generators/application/files/common-vite/.babelrc__tmpl__

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I think we can leave out the babel config unless the user wants to add it manually.